wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -211,7 +211,7 @@ addRecentPtxHistory wid currentHistory = do
   where
     toCandidates =
             txHistoryListToMap
-        .   mapMaybe (ptxPoolInfo . _ptxCond)
+        .   mapMaybe (preview _PtxApplying . _ptxCond)
```

Comment:
Can you explain this one more clearly? This is not just logging, this changes which txs are returned.

Reply:
Right, my bad /